readme.md

```diff
@@ -329,6 +339,24 @@ Default: `false`
 
 If `true`, no quoting or escaping of arguments is done on Windows. Ignored on other platforms. This is set to `true` automatically when the `shell` option is `true`.
 
+### child_process.kill([signal], [options])
+
+You may kill the child process with this function. It is slightly different that the original function.
+This one allows you to kill the process with `SIGTERM` and after a specified timeout (5 seconds) a `SIGKILL` is invoked to ensure killing the process.
+
+#### options.retry
+
+Type: `boolean`<br>
+Default: true
+
+Tells the kill to start a second attemp to kill.
```

Comment:
`Tells the kill to start` is a little confusing to me. Also "attempt" has a typo. 

Maybe instead: If the first signal does not terminate the child process, tries again using `SIGKILL`.